README.md

```diff
@@ -1,27 +1,60 @@
 # AvidCovider
+The project aims to support gathering daily information on the spread of the corona virus.
+The project based on an [Angular CLI](https://github.com/angular/angular-cli) version 7.1.3, offering a dynamic chatbot frontend service (named hatool) and a dynamic script. Another example for a live implementation can be found on [another repo](https://github.com/hasadna/reportit-user).
 
-This project was generated with [Angular CLI](https://github.com/angular/angular-cli) version 7.1.3.
 
-## Development server
+## Technical Background
+
+The site serves via [Github Pages](https://pages.github.com/)
```

Comment:
We're moving away from Github pages actually.

Reply:
@akariv wehre to?